src/policy/rbf.cpp

```diff
@@ -35,7 +36,7 @@ RBFTransactionState IsRBFOptIn(const CTransaction& tx, const CTxMemPool& pool)
 
     // If all the inputs have nSequence >= maxint-1, it still might be
     // signaled for RBF if any unconfirmed parents have signaled.
-    const CTxMemPoolEntry entry{*pool.mapTx.find(tx.GetHash())};
+    const CTxMemPoolEntry entry{**Assert(pool.GetIter(tx.GetHash()))};
```

Comment:
Are you assuming that the callers of `IsRBFOptIn` must pass a mempool transaction with the `Assert`?

Reply:
Yes, as I understand it that is what is checked in the code block above, so this should always hold.